src/main/java/org/folio/edge/sip2/MainVerticle.java

```diff
@@ -82,6 +86,28 @@ private String getSanitizedConfig() {
   public void start(Promise<Void> startFuture) {
     log.debug("Startup configuration: {}", this::getSanitizedConfig);
 
+    HttpServer httpServer = vertx.createHttpServer();
+
+    httpServer.requestHandler(request -> {
+      log.info("path : {}",request.path());
+      if (request.path().equals("/admin/health")) {
```

Comment:
Please add a constant for "/admin/health"

Reply:
Constant added.